webpack.config.js

```diff
@@ -1,105 +1,111 @@
-require('dotenv').config()
-const webpack = require('webpack')
-const path = require('path')
-const CreateFileWebpack = require('create-file-webpack')
-const MiniCssExtractPlugin = require('mini-css-extract-plugin')
-const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
-const WebpackShellPlugin = require('webpack-shell-plugin')
-const WriteFilePlugin = require('write-file-webpack-plugin')
-const LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
-const CircularDependencyPlugin = require('circular-dependency-plugin')
-const UnusedWebpackPlugin = require('unused-webpack-plugin')
-const _ = require('lodash')
+require("dotenv").config();
+const webpack = require("webpack");
+const path = require("path");
+const CreateFileWebpack = require("create-file-webpack");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const UglifyJSPlugin = require("uglifyjs-webpack-plugin");
+const WebpackShellPlugin = require("webpack-shell-plugin");
+const WriteFilePlugin = require("write-file-webpack-plugin");
+const LodashModuleReplacementPlugin = require("lodash-webpack-plugin");
+const CircularDependencyPlugin = require("circular-dependency-plugin");
+const UnusedWebpackPlugin = require("unused-webpack-plugin");
+const _ = require("lodash");
 
-const reduxLogMode = process.env.REDUX_LOGGING === 'VERBOSE' ? 'VERBOSE' : 'SILENT'
+const reduxLogMode =
+  process.env.REDUX_LOGGING === "VERBOSE" ? "VERBOSE" : "SILENT";
 
-const editorHtmlTemplate = require('./src/html-template.js')
-const evalFrameHtmlTemplate = require('./src/eval-frame/html-template.js')
+const editorHtmlTemplate = require("./src/html-template.js");
+const evalFrameHtmlTemplate = require("./src/eval-frame/html-template.js");
 
-const editorHtmlTemplateCompiler = _.template(editorHtmlTemplate)
-const evalFrameHtmlTemplateCompiler = _.template(evalFrameHtmlTemplate)
+const editorHtmlTemplateCompiler = _.template(editorHtmlTemplate);
+const evalFrameHtmlTemplateCompiler = _.template(evalFrameHtmlTemplate);
 
-const DEV_SERVER_PORT = 8000
+const DEV_SERVER_PORT = 8000;
 
-const BUILD_DIR = path.resolve(__dirname, 'build/')
-let APP_PATH_STRING
-let CSS_PATH_STRING
+const BUILD_DIR = path.resolve(__dirname, "build/");
+let APP_PATH_STRING;
+let CSS_PATH_STRING;
 
-let { EDITOR_ORIGIN } = process.env
-let { EVAL_FRAME_ORIGIN } = process.env
-const { USE_OPENIDC_AUTH } = process.env
-const { IODIDE_PUBLIC } = process.env || false
+let { EDITOR_ORIGIN } = process.env;
+let { EVAL_FRAME_ORIGIN } = process.env;
+const { USE_OPENIDC_AUTH } = process.env;
+const { IODIDE_PUBLIC } = process.env || false;
 
-const PYODIDE_VERSION = process.env.PYODIDE_VERSION || '0.4.0'
-process.env.PYODIDE_VERSION = PYODIDE_VERSION
+const PYODIDE_VERSION = process.env.PYODIDE_VERSION || "0.5.0";
```

Comment:
apologies - apparently no one had applied `prettier` on this file. You can ignore all the other changes except this one.